src/renderer/features/shared/components/library-header.tsx

```diff
@@ -64,7 +66,9 @@ export const LibraryHeader = forwardRef(
                     className={styles.background}
                     style={{ background, filter: `blur(${blur ?? 0}rem)` }}
                 />
-                <div className={styles.backgroundOverlay} />
+                <div
+                    className={`${styles.backgroundOverlay} ${(albumBackground && styles.opaqueOverlay) || ''}`}
+                />
```

Comment:
Can you use the `clsx` library for this instead? Search the repo for clsx for an example.

Reply:
Will do when I get the time again which might not be today possibly so unless this is a critical issue I wouldn't let it hold this from being merged

Reply:
All good, I just edited your branch from my side.